dockerfiles/theia/push-cdn-files-to-akamai.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
+base_dir=$(cd "$(dirname "$0")"; pwd)
+current_dir=$(pwd)
+if [ "${AKAMAI_CHE_AUTH:-}" != "" ]; then
```

Comment:
shouldn't we use  `-z` or `-n` internal bash flag to test empty or not empty strings ?